wingtips-core/src/main/java/com/nike/wingtips/TraceAndSpanIdGenerator.java

```diff
@@ -77,4 +113,70 @@ protected static Random getRandomInstance(String desiredSecureRandomImplementati
 
         return randomToUse;
     }
+
+    /**
+     * <p>
+     *     The code in this class came from the Zipkin repository v1.7.0
+     *     (https://github.com/openzipkin/zipkin/blob/master/zipkin/src/main/java/zipkin/internal/Util.java)
+     *     and licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0).
+     * </p>
+     */
+    protected static class ZipkinHexHelpers {
```

Comment:
My personal habit - I don't like private as a general rule because there have been too many fist-shaking moments using other people's libraries where I want to reuse an entire class except one stupid little private method that I wish I could override (for example). Protected also tends to make unit testing a little easier.

I agree in this case with all the static-ness it's far fetched there would ever be any benefit from making it protected instead of private. But it also shouldn't hurt, so habit wins I guess.